salt/utils/pyobjects.py

```diff
@@ -332,6 +330,23 @@ def __set_attributes__(cls):
 
                 match_groups[grain][match][name] = filt.__dict__[name]
 
+        if hasattr(cls, 'priority'):
```

Comment:
I had considered that, but I wanted to make sure the priority had an `__iter__` attribute since you can iterate over a string using a for loop. I wanted `priority = 'os_family'` to be considered invalid and to log an error.

There is definitely some room for improvement though. I just pushed another commit, let me know what you think.